docs/contribute.html

```diff
@@ -0,0 +1,114 @@
+<!-- contribute page created by James Languirand for use with RaiderPlanner project for CEG-3120-01, WSU, Fall 2017 -->
+<!DOCTYPE html>
+<html lang="en">
+  <!-- Head Information -->
+  <head>
+	<title>Contribute to the RaiderPlanner</title>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+    <meta name="description" content="Informational page about how you can contribute to the RaiderPlanner and licensing information.">
+	<link rel="shortcut icon" href="images/favicon.png">
+	
+    <!-- Bootstrap core JavaScript -->
+    <script src="vendor/jquery/jquery.min.js"></script>
+    <script src="vendor/popper/popper.min.js"></script>
+    <script src="vendor/bootstrap/js/bootstrap.min.js"></script>
+    
+    <!-- Bootstrap core CSS -->
+    <link href="vendor/bootstrap/css/bootstrap.min.css" rel="stylesheet">
+    <link href="css/one-page-wonder.css" rel="stylesheet">
+	
+	<!-- CSS Created by James Languirand -->
+	<link href="css/about.css" rel="stylesheet">
+  </head>
+  <body>
+    <!-- Navigation -->
+    <nav class="navbar navbar-expand-lg navbar-dark bg-dark fixed-top">
+      <div class="container">
+      <a href="index.html"><img src="images/test.png" style="width:100px;height:75px;"></a>
+        <a class="navbar-brand" href="index.html">Raider Planner</a>
+        <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarResponsive" aria-controls="navbarResponsive" aria-expanded="false" aria-label="Toggle navigation">
+          <span class="navbar-toggler-icon"></span>
+        </button>
+        <div class="collapse navbar-collapse" id="navbarResponsive">
+          <ul class="navbar-nav ml-auto">
+            <li class="nav-item">
+              <a class="nav-link" href="index.html">Home</a>
+            </li>
+            <li class="nav-item">
+              <a class="nav-link" href="about.html">About</a>
+            </li>
+            <li class="nav-item active">
+              <a class="nav-link" href="contribute.html">Contribute
+                <span class="sr-only">(current)</span>
+              </a>
+            </li>
+            <li class="nav-item">
+              <a class="nav-link" href="download.html">Download</a>
+            </li>
+          </ul>
+        </div>
+      </div>
+    </nav>
+    
+	<!-- Text Right, Image Left -->
+    <section>
+      <div class="container">
+        <div class="row align-items-center">
+          <div class="col-md-6">
+            <div class="p-5">
+              <img class="img-fluid" src="images/code.jpg" alt="Code Image">
```

Comment:
The src attributed refers to `code.jpg`, but the file that was committed is called `code.JPG`.  On windows this does not make a difference because its file systems are case-insensitive.  However, on Linux (which is what the site will be served from at GitHub) that results in this:

![screenshot from 2017-10-03 06-58-44](https://user-images.githubusercontent.com/9941546/31122312-48fcf322-a809-11e7-8e82-27c87a3f6a19.png)

The correct resolution is to rename the file to be `code.jpg`.